### DISCUSSION TITLE
Theia IDE 1.55 preview testing

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#14394] [Release Announcements] [answered]
The new version 1.55.0 of the Theia IDE is available on the preview channel now, please join the preview testing! You can download it here: [Linux](https://download.eclipse.org/theia/ide-preview/1.55.0/linux/TheiaIDE.AppImage), [Mac](https://download.eclipse.org/theia/ide-preview/1.55.0/macos/TheiaIDE.dmg), [Windows](https://download.eclipse.org/theia/ide-preview/1.55.0/windows/TheiaIDESetup.exe). You can update your existing installation by setting the preference `updates.channel` to `preview`. Please respond here when you were able to test the preview without finding blockers, by commenting with a :heavy_check_mark: . If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.

**Comment by JonasHelming** [comment]
✔️

**Comment by sgraband** [comment]
Haven't found any new issues/blockers :heavy_check_mark:

**Comment by tsmaeder** [comment]
For me, as well, no new blockers, but newer versions of extensions like gitlens seem to work less well, something we should have a look at soon. But for me 1.55.0 is at least no worse than 1.54.0.

**Comment by jfaltermeier** [comment]
✔️

**Comment by tsmaeder** [comment]
Looks like everyone is happy with 1.55.0 then? So let's flip the switch, @jfaltermeier!

**Reply by jfaltermeier** [reply, marked as answer]
1.55.0 is promoted to the stable channel
